Portfolio setup and mobile version skeleton

myelin0 opened the pull request:
Add Toolbar and Headline for main mobile page :
index.html
style.css

myelin0 commented:
**Portfolio project setup and mobile skeleton**

Add Toolbar and Headline for main mobile page
- index.html
- style.css

lily-coder commented on README.md:
```diff
@@ -1 +1,39 @@
+
+- GitHub: [@myelin0](https://github.com/myelin0)
+- Twitter: [@Amy_Albania](https://twitter.com/Amy_albania)
+- LinkedIn: [@Emirjeta Veisllari](www.linkedin.com/in/emirjeta-veisllari-aa5b71222)
```
- [ ] [OPTIONAL] Please ensure all links in your README file are working. This is very important as it exudes professionalism.

lily-coder commented on index.html:
```diff
@@ -0,0 +1,33 @@
+    
+    <header class="nav-bar">
+           <a class="logo" href="#">My logo</a>
+           <a class="menu-button" href="#"><img src="/images/Menu.png" alt=""></a>
```
- [ ] [REQUIRED] Please ensure you stick to the design as much as possible. The hamburger button is missing on the right. So please ensure you add it and it should be in a `nav` tag, not an `a` tag

lily-coder commented on index.html:
```diff
@@ -0,0 +1,33 @@
+    </header>
+
+    <section class="headline">
+      <h2> I’m Emirjeta <br> Glad to see <br>you!</h2>
```
- [ ] [REQUIRED] Your main title is currently an h2 tag. please change this main heading of the page to the h1 tag. This is a part of good SEO.

lily-coder reviewed (changes requested):
Hi @myelin0 , 

Good job so far!🎉🎉🎉

To Highlight! 🟢
✔️ Good commit messages
✔️ Good job with the linter checks
✔️ Correct Github flow was used

There are some issues that you still need to work on to go to the next project but you are almost there!
![image](https://user-images.githubusercontent.com/63964646/143551191-c0e33599-a97b-46fb-91a5-267e663d3f5c.png)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

myelin0 commented:
**Fixed errors after review**

- Placed hamburger menu in nav tag.
- Changed the main title from h2 to h1
- Corrected Readme file link for Linkedin.

andersonlebon commented on style.css:
```diff
@@ -0,0 +1,105 @@
+  display: flex;
+  flex-direction: column;
+  overflow-y: scroll;
+  -webkit-overflow-scrolling: touch;
```
## [OPTIONAL]
- [ ] CONGRATULATIONS :clap: I  can see you follow the requirement of the first reviewer by putting the hamburger button in the `<nav>` tag and not in a `<a>` tag. However, you still need to stick to the design as much as possible as she said.
- Try to compare the original design on the left side and yours on the right side. Then improve your code by removing the `overflow-y: scroll` property or changing it with `overflow-y: hidden`. I think it would be nice also if you follow the same `font-family`, `font-weight`, and the `font-size` in the original design as the requirements said.
![Screenshot from 2021-11-26 11-50-09](https://user-images.githubusercontent.com/65068771/143563294-8e9d5bf6-ccd8-40eb-99d7-37c12f72e60f.png)

andersonlebon reviewed (approved):
Hi @myelin0 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

![nodding-yes](https://user-images.githubusercontent.com/65068771/143095390-344d1cb0-b853-4b91-8662-5d9049dc9e60.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHubflow :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 
- Your design is looking great :+1:  :100: :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
